### PR TITLE
Remove duplicate helper functions in cache tests.

### PR DESCRIPTION
**Pull request by fahimfaisaal**
# Welcome to the JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?
- [x] Added new **testerFunction** file

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [ ] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Cache/test/TesterFunction.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/**
```
Comment (raklaptudirm) [inline Cache/test/TesterFunction.js:1]: Put these in the same file as the tests.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

File: Cache/test/TesterFunction.js (hunk 1 above)
Comment (fahimfaisaal) [reply Cache/test/TesterFunction.js:1]: you mean `TesterFunction.test.js`

Comment (raklaptudirm) [reply Cache/test/TesterFunction.js:1]: No, there is no need for a separate file for those functions, as they will probably not be reused.

Comment (fahimfaisaal) [reply Cache/test/TesterFunction.js:1]: `FibonacciCache` used on two files, 
1. `LFUCache.test.js`
2. `LRUCache.test.ts`

Comment (raklaptudirm) [reply Cache/test/TesterFunction.js:1]: Then rename the file to something more specific than `TesterFunction.js`.

Comment (fahimfaisaal) [reply Cache/test/TesterFunction.js:1]: Could you prefer `utils.js`?

Comment (fahimfaisaal) [reply Cache/test/TesterFunction.js:1]: Would you prefer `utils.js`?

Comment (raklaptudirm) [reply Cache/test/TesterFunction.js:1]: No, something like `cacheTest.js`. Something that is relevant to the function.

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**